spyderlib/plugins/ipythonconsole.py

```diff
@@ -180,12 +176,22 @@ def setup_page(self):
 
         interface_layout = QVBoxLayout()
         interface_layout.addWidget(banner_box)
-        interface_layout.addWidget(gui_comp_box)
         interface_layout.addWidget(pager_box)
         interface_layout.addWidget(calltips_box)
         interface_layout.addWidget(ask_box)
         interface_group.setLayout(interface_layout)
 
+        comp_group = QGroupBox(_("Completion Widget"))
+        comp_label = QLabel(_("Decide what type of completion to use"))
+        comp_label.setWordWrap(True)
+        completers = [("plain", 0), ("droplist", 1), ("ncurses", 2)]

```

Comment:
The default (as before) should be `droplist`, not `plain`. This is to match the same behavior as in the Editor and Python consoles (i.e. to use a completion widget :-)